cloud/datasync/src/index.ts

```diff
@@ -0,0 +1,145 @@
+import * as Promise from 'bluebird';
+import * as Logger from 'bunyan';
+import * as Express from 'express';
+import SyncGlobalOptions from './syncOptions';
```

Comment:
probably need a rebase so 01636ff branches off of master instead of the sync branch